Sok-O-Vando/src/main/java/org/academiadecodigo/asynctomatics/sokovando/Game.java

```diff
@@ -25,7 +27,7 @@
     Picture levelClear;
     Picture levelClear600;
     Picture win;
-    boolean started = false;
+    public boolean started = false;
```

Comment:
Yes, made it public because one class that uses it was moved to a package. There is a ton of properties with no explicit (or default) access modifier. But It is material for another commit/refactor stage.